src/components/Contacts/ContactFlow/ContactFlow.tsx

```diff
@@ -51,6 +52,7 @@ export const ContactFlow: React.FC<Props> = ({
 
   const { t } = useTranslation();
   const { getContactStatusesByPhase } = useContactPartnershipStatuses();
+  const phases = useApiConstants()?.phases;
```

Comment:
Same here with writing ::
```suggestion
  const {phases} = useApiConstants();
```